tasks/cloudwatch.yml

```diff
@@ -1,8 +1,11 @@
 ---
 - name: Install cloudwatch plugin
   gem:
-    name: "fluent-plugin-cloudwatch-logs"
+    name: "{{ item }}"
     executable: /opt/td-agent/embedded/bin/fluent-gem
-    state: present
+    state: "{{ fluentd_plugins_state }}"
     user_install: false
   notify: restart td-agent
+  with_items:
```

Comment:
we can't use the `fluentd_plugins` ?

Reply:
using fluentd_plugins means we have to update all users variables and set state: latest for every plugin :/

I prefer to limit the impact by only put latest on aws core.
Since it is a dependancy for CW, it make sense to have it in the same dependancy file

Reply:
ah yes. you're right